packages/frontend/src/components/common/NavMenuItem.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC } from 'react';
+import { Route, useHistory } from 'react-router-dom';
+import { MenuItem } from '@blueprintjs/core';
+
+interface Props {
+    to: string;
+    exact?: boolean;
+}
+
+const NavMenuItem: FC<Props & React.ComponentProps<typeof MenuItem>> = ({
```

Comment:
Can we rename to `SidebarMenuItem`? I was initially confused that this was for the `AppBar` until I saw the usage further down

Reply:
For context, I named it after the 'NavLink' from 'react-router'.
I don't want to name it after sidebar since it can be used anywhere and there might be sidebar menus that don't depend on react router. 
Do you prefer 'RouterMenuItem' ?

Reply:
Ah I see because it's unstyled. `RouterMenuItem` is good!